proto/check.proto

```diff
@@ -33,8 +33,8 @@ message Policy {
 }
 
 message HeaderPolicy {
-  bytes minmum_qe_svn = 1;   // should be 2 bytes
-  bytes minimum_pce_svn = 2;  // should be 2 bytes
+  uint32 minmum_qe_svn = 1; 
```

Comment:
Comment that the values should not exceed uint16 max

Reply:
Done.